docs/sphinx/specification/cudaq/namespace.rst

```diff
@@ -3,9 +3,14 @@ Namespace and Standard
 
 **[1]** The CUDA-Q language is a library-based extension for C++. As such, all
 pertinent data structures and library functions exist within the CUDA-Q
-namespace. This namespace is denoted :code:`::cudaq`. 
+namespace. This namespace is denoted :code:`::cudaq`.
 
-**[2]** CUDA-Q is a C++ library-based language extension adherent to the `C++20 <https://en.cppreference.com/w/cpp/20>`_
+**[2]** The :code:`cudaq::contrib` namespace contains features contributed by
```

Comment:
My suggested phrasing: 

The :code:`cudaq::contrib` namespace contains features contributed by the larger community. While these features provide useful functionality and are maintained across releases, they may be more limited in functionality and may not cover some scenarios. If you encounter issues or would like to request more extensive features, please file a GitHub issue and we will consider them for a future release.